src/session.jl

```diff
@@ -6,26 +6,35 @@ module session
 export GoogleSession, authorize
 
 import Base: string, print, show
-using Dates, Base64 
+using Dates, Base64
 
 #using Requests
-using HTTP, HTTP.Messages 
+using HTTP, HTTP.Messages
 
-import JSON
-import MbedTLS
+using JSON: JSON
+using MbedTLS: MbedTLS
 
 using ..error
 using ..credentials
 using ..root
 
+global mbedtlslock = ReentrantLock()
```

Comment:
```suggestion
const MBEDTLSLOCK = ReentrantLock()
```
`global` is used within functions to make a variable refer to a global definition instead of local.